docs/production/export-and-import.md

```diff
@@ -13,7 +13,7 @@ service (or back):
     version (most precisely, one where `manage.py showmigrations` has
     the same output).
   - Backups must be restored on a server running the same PostgreSQL
-    version.
+    version.  To use the same version of PostgreSQL that the backup was taken on, pass --postgresql-version=["your version"]
```

Comment:
I think it'd be better to include this text further down in the document, where we document the actual command.